dash.js

```diff
@@ -78,6 +78,28 @@ const MyDashActor = new Lang.Class({
         this.actor.connect('allocate', Lang.bind(this, this._allocate));
 
         this.actor._delegate = this;
+
+        let alignment = 0.25;
+        if (this._isHorizontal)
+            alignment = 0.5;
+        // Popup menu
+        this.menu = new PopupMenu.PopupMenu(this.actor, alignment, this._position);
+        this.menu.actor.hide();
+        this.menu.actor.add_style_class_name('panel-menu');
```

Comment:
Skipping this css class seems to fix the issue with the additional spacing between the menu and the dock when on bottom. Why do we need this class?